packages/extension-points/README.md

```diff
@@ -35,3 +35,14 @@ widgets:
     component: js/build/options_buttons.widget.js
     multiple: true
 ```
+
+### Routes Configuration
```

Comment:
Should we document that for  menu links they can still provide them as normal menu link inside a drupal module?

Reply:
Do we have any documentation about menu links already?